scripts/release.js

```diff
@@ -64,8 +64,8 @@ async function main() {
     console.log();
     console.log(`  👉 yarn release --finalize ${toVersion} ${pullRequestId}`);
   } else {
-    const toVersion = process.argv[4];
-    const pullRequestId = process.argv[5];
+    const toVersion = process.argv[3];
+    const pullRequestId = process.argv[4];
```

Comment:
Could you flip the `if` statement so `if (process.argv[2] === 'finalize')` immediately precedes these lines? This should make it clearer what is going on and prevent us from making similar mistakes again.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1536/commits/7fb7235a244b1c7f5dc9869e962da4c5d77e5fd6